packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCountry.tsx

```diff
@@ -2,84 +2,69 @@ import { Control, Controller } from 'react-hook-form';
 
 import styled from 'styled-components';
 
-import { Flag, Select } from '@trezor/components';
-import { borders, spacingsPx } from '@trezor/theme';
+import { Flag, Select, Row } from '@trezor/components';
+import { borders, spacings } from '@trezor/theme';
 
 import regional from 'src/constants/wallet/coinmarket/regional';
 import { CountryOption } from 'src/types/wallet/coinmarketCommonTypes';
 import { getCountryLabelParts } from 'src/utils/wallet/coinmarket/coinmarketUtils';
 import { useCoinmarketFormContext } from 'src/hooks/wallet/coinmarket/form/useCoinmarketCommonForm';
-import { CoinmarketFormInputLabel } from 'src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputLabel';
 import {
     CoinmarketBuySellFormProps,
     CoinmarketFormInputDefaultProps,
 } from 'src/types/coinmarket/coinmarketForm';
-import { CoinmarketFormOption, CoinmarketFormOptionLabel } from 'src/views/wallet/coinmarket';
 import { FORM_COUNTRY_SELECT } from 'src/constants/wallet/coinmarket/form';
 import { CoinmarketTradeBuySellType } from 'src/types/coinmarket/coinmarket';
+import { Translation } from 'src/components/suite';
+import { TranslationKey } from 'src/components/suite/Translation';
 
-const FlagContainer = styled.div`
-    position: relative;
+const FlagWrapper = styled.div`
```

Comment:
Does not look pretty, I know about your concerns. Maybe an extension of the flag component would be better?
![obrazek](https://github.com/user-attachments/assets/2dc9375e-32b2-4e2f-914c-598e09672828)

Reply:
Will fix this.

Reply:
@adamhavel can we please remove the rounding of flags? Flags are mostly rectangle

Reply:
![obrazek](https://github.com/user-attachments/assets/85f860f4-8ba4-490b-8358-039bc8427b3d)

Reply:
> @tomasklim would be nicer to synchronize it with a crypto currency/token design IMO, but it is ok.
flags are mostly rectangles/squares
coins are mostly circles (but not all, we are also cutting and destroying some logos)